diff --git a/node_modules/react-native-maps/react-native-google-maps.podspec b/node_modules/react-native-maps/react-native-google-maps.podspec
index 14417b4..077da24 100644
--- a/node_modules/react-native-maps/react-native-google-maps.podspec
+++ b/node_modules/react-native-maps/react-native-google-maps.podspec
@@ -18,5 +18,5 @@ Pod::Spec.new do |s|
 
   s.dependency 'React'
   s.dependency 'GoogleMaps', '3.2.0'
-  s.dependency 'Google-Maps-iOS-Utils', '2.1.0'
+  s.dependency 'Google-Maps-iOS-Utils', '3.2.0'
 end
